File: Task-2.2.py
string = input('Введите строку ')
posX = string.find("x")
posW = string.find("w")
if posW == -1 or posX == -1:
    print('Нет таких символов')
else:
    if posX < posW:
        print('Первее X')
    else:
        if posW < posX:
            print('Первее W')

File: task_2_2.pas
program task_2_2;

var
  str:string;
  posX,posW:integer;
  
begin
  writeln('Введите строку');
  readln(str);
  posX:= Pos('x',str);
  posW:= Pos('w',str);
  if (posW = 0) or (posX = 0) then
    writeln('Нет таких символов')
  else
    if posX < posW then
      writeln('Первее X')
    else
      writeln('Первее W');
end.